cmake/cprConfig.cmake.in

```diff
@@ -2,6 +2,7 @@ include(CMakeFindDependencyMacro)
 @PACKAGE_INIT@
 
 find_dependency(CURL REQUIRED)
+find_dependency(OpenSSL REQUIRED)
```

Comment:
But what happens in case a user does not use OpenSSL and instead Windows SSL or mbedtls? To me this should be conditional.

Reply:
Fixed